src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -1100,6 +1105,15 @@ object Parser {
       case left ~ right => PMultisetConversion(PRangeSequence(left, right).range(left, right))
     }
 
+    lazy val optionNone : Parser[POptionNone] =
+      "none" ~> ("[" ~> typ <~ "]") ^^ POptionNone
+
+    lazy val optionSome : Parser[POptionSome] =
+      "some" ~> ("(" ~> expression <~ ")") ^^ POptionSome
+
+    lazy val optionGet : Parser[POptionGet] =
+      "option" ~> ("(" ~> expression <~ ")") ^^ POptionGet
```

Comment:
can you change the name before it is merged in? You could use the, get, option-get, etc

Reply:
Yes, fixed it. It now uses "get".